vta/hardware/chisel/Makefile

```diff
@@ -15,6 +15,8 @@
 # specific language governing permissions and limitations
 # under the License.
 
+export VERILATOR_INC_DIR = /usr/local/share/verilator/include
```

Comment:
This should be external to the Makefile IMO; and set by the user. See: https://docs.tvm.ai/install/from_source.html#tvm-package

I think we should make it clear in the instructions that this variable needs to be set by the user, when they install verilator.

Reply:
maybe just wrap the export in an `ifndef`

Reply:
I see. I guess we could try to check if the path exists in the makefile then?.

I think having to setup  more environment variables is a pain for the user.

Reply:
I think that hard coded paths will fail at some point depending on os, or package manager. Given that the user will need to install verilator themselves, this is part of the installation process. We just need to add a line saying to export blah.

Reply:
Do you want me to add a comment in the Makefile in the meantime?

Reply:
we should have a cascade of checks: check that the variable exists, if not, set to default path; check that the binary exists, else error out

Reply:
Alright, I just added the cascade of checks for binary first then for the path.

Reply:
Thanks, it looks good except that we shouldn't use `export` in a Makefile. Instead can we use a `?=` to set if not set?

Reply:
Done